index.js

```diff
@@ -189,7 +189,7 @@ module.exports = {
     await appBlueprint.install(appOptions);
 
     await Promise.all([
-      this.updateTestAppPackageJson(path.join(testAppPath, 'package.json'), isPnpm(options)),
+      this.updateTestAppPackageJson(path.join(testAppPath, 'package.json'), isPnpm(options), options),
```

Comment:
why the extra arg at the end?

Reply:
Left that on accident when I was trying something different 😅